xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -80,11 +81,12 @@ enum UserProfileDetailNotification: Hashable {
 /// profile's internal state.
 struct UserProfileDetailDescription: Hashable {
     var address: Slashlink
+    var user: UserProfile?
     var initialTabIndex: Int = UserProfileDetailModel.recentEntriesTabIndex
 }
 
 enum UserProfileDetailAction {
-    case appear(Slashlink, Int)
+    case appear(Slashlink, Int, UserProfile?)
```

Comment:
Pass basic user information in action if we have it, allows for loading state to render a partial view.